Content.Server/Silicons/Borgs/BorgSystem.Modules.cs

```diff
@@ -274,6 +274,23 @@ public bool CanInsertModule(EntityUid uid, EntityUid module, BorgChassisComponen
             return false;
         }
 
+        foreach (var containedModuleUid in component.ModuleContainer.ContainedEntities)
+        {
+            if (!TryComp<ItemBorgModuleComponent>(containedModuleUid, out var containedItemModuleComp)
+                || !TryComp<ItemBorgModuleComponent>(module, out var itemModuleComp))
+                continue;
+
+            for (int i = 0; i < itemModuleComp.Items.Count; i++)
+            {
+                if (itemModuleComp.Items[i] != containedItemModuleComp.Items[i])
+                    continue;
+            }
+
```

Comment:
This is going to spam TryComp on module constantly in the loop so please move this out of it.

Reply:
right